src/files.jl

```diff
@@ -20,10 +20,10 @@ function make_manifest(data_path::String, r::Regex; verbose::Bool = true)
       @error("Did not find any files in ", data_path ,".")
    end
 
-   results = map(fn->match(r"(\d{8})[T_](\d{6})", last(split(fn,'/')) ), df_files.Filename)
-   @assert all(map(r->length(r.captures),results).==2)
-   df_files.date_str = map(r->String(r.captures[1]),results)
-   df_files.time_str = map(r->String(r.captures[2]),results)
+   #results = map(fn->match(r"(\d{8})[T_](\d{6})", last(split(fn,'/')) ), df_files.Filename)
```

Comment:
Since this is shared across instruments, removing these will result in the manifets for NEID solar observations not having these fields.  We should be able to compensate by moving these lines to where the manifest is being read in NeidSolarScripts.